packages/devextreme/testing/tests/DevExpress.ui.widgets/tabs.markup.tests.js

```diff
@@ -129,14 +129,16 @@ QUnit.module('Aria accessibility', {
             QUnit.test(`3 items, repaintChangesOnly: ${repaintChangesOnly}, use: ${sourceName}`, function() {
                 helper.createWidget({ [`${sourceName}`]: this.items, repaintChangesOnly });
 
-                helper.checkAttributes(helper.$widget, { role: 'tablist', tabindex: '0' }, 'widget');
+                helper.checkAttributes(helper.$widget, { tabindex: '0' }, 'widget');
+                helper.checkAttributes(helper.widget._$wrapper, { role: 'tablist' }, 'tabsWrapper');
                 helper.checkItemsAttributes([], { attributes: ['aria-selected'], role: 'tab' });
             });
 
             QUnit.test(`[item1], add new item2, repaintChangesOnly: ${repaintChangesOnly}, use: ${sourceName}`, function(assert) {
                 helper.createWidget({ [`${sourceName}`]: [{ text: 'Item_1' }], repaintChangesOnly });
 
-                helper.checkAttributes(helper.$widget, { role: 'tablist', tabindex: '0' }, 'widget');
+                helper.checkAttributes(helper.$widget, { tabindex: '0' }, 'widget');
+                helper.checkAttributes(helper.widget._$wrapper, { role: 'tablist' }, 'tabsWrapper');
```

Comment:
Can we get rid of using of a private API?

Reply:
done